nusantara/nusa_datasets/wrete/wrete.py

```diff
@@ -0,0 +1,139 @@
+from pathlib import Path
+from sys import stdout
+from tempfile import tempdir
+from typing import List
+from unicodedata import category
+
+import datasets
+import pandas as pd
+
+from nusantara.utils import schemas
+from nusantara.utils.configs import NusantaraConfig
+from nusantara.utils.constants import Tasks, DEFAULT_SOURCE_VIEW_NAME, DEFAULT_NUSANTARA_VIEW_NAME
+
+_DATASETNAME = "wrete"
+_SOURCE_VIEW_NAME = DEFAULT_SOURCE_VIEW_NAME
+_UNIFIED_VIEW_NAME = DEFAULT_NUSANTARA_VIEW_NAME
+
+_LANGUAGES = ["ind"]  # We follow ISO639-3 language code (https://iso639-3.sil.org/code_tables/639/data)
+_LOCAL = False
+_CITATION = """\
+@INPROCEEDINGS{8904199,
+    author={Purwarianti, Ayu and Crisdayanti, Ida Ayu Putu Ari},
+    booktitle={2019 International Conference of Advanced Informatics: Concepts, Theory and Applications (ICAICTA)},
+    title={Improving Bi-LSTM Performance for Indonesian Sentiment Analysis Using Paragraph Vector},
+    year={2019},
+    pages={1-5},
+    doi={10.1109/ICAICTA.2019.8904199}
+}
```

Comment:
To be consistent with other dataset in a benchmark, could you also put a citation to IndoNLU here?